src/coreclr/tools/Common/JitInterface/CorInfoTypes.cs

```diff
@@ -1204,6 +1204,46 @@ public struct SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR
         public byte eightByteOffsets1;
     };
 
+    // StructFloadFieldInfoFlags: used on LoongArch64 architecture by `getLoongArch64PassStructInRegisterFlags` API
+    // to convey struct argument passing information.
+    //
+    // `STRUCT_NO_FLOAT_FIELD` means structs are not passed using the float register(s).
+    //
+    // Otherwise, and only for structs with no more than two fields and a total struct size no larger
+    // than two pointers:
+    //
+    // The lowest four bits denote the floating-point info:
+    //   bit 0: `1` means there is only one float or double field within the struct.
+    //   bit 1: `1` means only the first field is floating-point type.
+    //   bit 2: `1` means only the second field is floating-point type.
+    //   bit 3: `1` means the two fields are both floating-point type.
+    // The bits[5:4] denoting whether the field size is 8-bytes:
+    //   bit 4: `1` means the first field's size is 8.
+    //   bit 5: `1` means the second field's size is 8.
+    //
+    // Note that bit 0 and 3 cannot both be set.
+    public enum LAFlags
```

Comment:
This should match the name in corinfo.h

Reply:
Ok,thanks.
I will do it tomorrow morning.